### DISCUSSION TITLE
analogWrite at 841

### DISCUSSION THREAD

**Discussion by bohumir-brocko** [SpenceKonde/ATTinyCore#515] [General]
In the Arduino discussion, I found a mention of PWM on pins. I'm solving the same problem. I need 4 PWM outputs and so far the pins I have tried behave like digitalWrite. What should I do to solve this? My analogWrite does not work as expected.

**Comment by bohumir-brocko** [comment]
I found 4 pins that work. I won't need the rest. :)

**Comment by SpenceKonde** [comment]
We do have a pinout chart on the ATTiny441/841 doc page: https://github.com/SpenceKonde/ATTinyCore/blob/master/avr/extras/ATtiny_x41.md which covers all the pin functionality, the two options for the order the pins are numbered in, and describes any particular features  that you should be aware of, or particular "gotcha's" with the parts that surprise people, I encourage everyone to read the part specific docs for the parts you're using - there's a lot of useful  basic info there,
